### DISCUSSION TITLE
反华分子能不能414啊，别在这恶心人！

### DISCUSSION THREAD

**Discussion by zhangf115** [WeNeedHome/SummaryOfLoanSuspension#591] [Show and tell]
论坛被这些反华分子搞的乌烟瘴气的，管理员大人快清一清吧，要么就干脆关闭发帖权限，这里已经被一些别有用心的人发到外网去了，引来了大量境外敌对分子，借机大做文章，恶意把问题上升到国家层面，以此攻击抹黑党和国家！
也奉劝那些拿着反华经费的汉奸们，不管是断贷还是什么，都轮不到你们这些境外人士指手画脚见缝插针，你们在这煽动情绪制造矛盾，只会让我们更加厌恶！滚回你们霉国爹那里去吧！去享受你们的疫情和枪击！

**Comment by C-Joey** [comment]
厉害 直钩钓鱼

**Reply by ethenlong** [reply]
难道是友军？

**Reply by tanuk1997** [reply]
> 难道是友军？

一眼丁真，模仿跪族喊青天大老爷呢

**Comment by dongyc519** [comment]
这种帖子需要被删除。

**Reply by Mmmmeruem** [reply]
为什么要删除，如果删除的话，在GITHUB跟在国内的网站还有什么区别呢？都只能发出一种声音？

**Reply by Mmmmeruem** [reply]
自由讨论的话，这些问题同样可以自由讨论

**Reply by taopeng-life** [reply]
说事实就要删除，社么鬼逻辑。房子都没了，还怕丢人丢到国外去？github也要学国内的一堆APP各种审查？幸福就对了？

**Comment by user-hook-name** [comment]
这就是国内为啥什么都要审核,干什么事情都要实名制的逻辑.
排除异己,非黑即白.

**Reply by a6749920** [reply]
你不知道干部晋升党员优先吗，党员才是我的人，天天给你开会

**Comment by yalo-ddaa** [comment]
这里就是外网，话说为啥互联网会有“内外网”之分？

**Reply by redapple2021** [reply]
有些人跪久了，没有自己的思想，喜欢主动替主子说话，自我阉割。
狭隘的民族主义情绪，什么年代的，现在只有文明人和野蛮人的分别，文明人走到哪里都受欢迎
愚昧无知野蛮的人哪里都不受欢迎
全球这么多国家，这几年有多少国家对中国的观感直线下降的

**Comment by Mmmmeruem** [comment]
你翻墙都犯法了，兄弟。

**Reply by Mmmmeruem** [reply]
马克思教我们的，用辩证的方法去看

**Comment by TALK1949** [comment]
我想这篇应该是真的对业主没啥帮助。

**Comment by allenyan513** [comment]
钓鱼贴

**Comment by JACWATA** [comment]
> 论坛被这些反华分子搞的乌烟瘴气的，管理员大人快清一清吧，要么就干脆关闭发帖权限，这里已经被一些别有用心的人发到外网去了，引来了大量境外敌对分子，借机大做文章，恶意把问题上升到国家层面，以此攻击抹黑党和国家！ 也奉劝那些拿着反华经费的汉奸们，不管是断贷还是什么，都轮不到你们这些境外人士指手画脚见缝插针，你们在这煽动情绪制造矛盾，只会让我们更加厌恶！滚回你们霉国爹那里去吧！去享受你们的疫情和枪击！

你行得正坐的端怕什么大做文章，搞笑

**Comment by songee2020** [comment]
该话题与项目关联性不大，且有“恶意引战”嫌疑。
建议停止，并提醒大家不要浪费注意力在该话题。

**Comment by JusticeEER** [comment]
> 论坛被这些反华分子搞的乌烟瘴气的，管理员大人快清一清吧，要么就干脆关闭发帖权限，这里已经被一些别有用心的人发到外网去了，引来了大量境外敌对分子，借机大做文章，恶意把问题上升到国家层面，以此攻击抹黑党和国家！ 也奉劝那些拿着反华经费的汉奸们，不管是断贷还是什么，都轮不到你们这些境外人士指手画脚见缝插针，你们在这煽动情绪制造矛盾，只会让我们更加厌恶！滚回你们霉国爹那里去吧！去享受你们的疫情和枪击！

反美分子死妈了，回去吃蝙蝠吧

**Reply by sqpk** [reply]
各地烂尾楼是假的吗？有些楼盘烂尾几年是假的吗？政府监管不力是假的？当地政府不是党吗？国内各种封杀是假的吗？710是假的吗？他们是如何处理的？公布人民了吗？他们怕人民吗？

**Comment by RE67** [comment]
一眼粉蛆，鉴定为纯纯的韭菜。

**Reply by wangsir003** [reply]
你怎么用汉字了，你个汉奸

**Comment by sqpk** [comment]
你他么能不能414啊，你说烂尾不是政府的错他么的还是购房者的错，既然政府连蹲着吃饭就要发文管，钱收了地卖了就不管购房者的死活了是吧，有些烂尾甚至好几年了监管干什么吃的，现在说两句不好听的国内能发出去吗，另外百万烂尾楼不就应该上升到国家层面让你处理？现在都翻到外面了，指责几句就扣个反华的帽子，正是有你们的无脑维护，有些官员才可以肆无忌惮为所欲为！

**Reply by wen-EO** [reply]
说的好

**Comment by taopeng-life** [comment]
@zhangf115  你怎么不去死啊，你说烂尾不是政府的错，还他么的还是购房者的错

**Comment by cccp057** [comment]
没翻墙啊，我没连外网正常进来的啊。
也不要扯什么政治问题，他骂国家你可以骂回去啊，我看有几个帖子那些闹事的就被骂回去了嘛

**Comment by BobbyYang9** [comment]
民主的特点是允许一切声音的发出，哪怕是反民主的言论

**Reply by Liboce** [reply]
民主是你可以批评公权力而不会受到惩罚

**Comment by Universe-Wong** [comment]
710，太子爷，停贷声明。这几天这几件事还不够你认识吗？sb5m

**Comment by fanren112233** [comment]
说不得不好 说共产党不好就是境外势力 你能不能死一死啊

**Comment by a6749920** [comment]
要求太高了，境外势力要求会git，要上github

**Comment by radiateboy** [comment]
口舌之争，能掀起什么大浪？都是在网上装大爷，回家跟孙子一样。

**Comment by uueer** [comment]
脑子不好，赶紧去找你习爹给你治一治。在中国呆久了，都被奴役的思想都没了。 真脑残一个

**Comment by hktk1024** [comment]
一时分不清是敌军还是友军

**Comment by xiaoheikejj** [comment]
> 论坛被这些反华分子搞的乌烟瘴气的，管理员大人快清一清吧，要么就干脆关闭发帖权限，这里已经被一些别有用心的人发到外网去了，引来了大量境外敌对分子，借机大做文章，恶意把问题上升到国家层面，以此攻击抹黑党和国家！ 也奉劝那些拿着反华经费的汉奸们，不管是断贷还是什么，都轮不到你们这些境外人士指手画脚见缝插针，你们在这煽动情绪制造矛盾，只会让我们更加厌恶！滚回你们霉国爹那里去吧！去享受你们的疫情和枪击！

你反正有房子住，可以随意在这指点江山，如果你也是烂尾楼业主你还能在这指点江山我就服你

**Comment by mand-byte** [comment]


**Comment by a805883237** [comment]


**Reply by seamasZ** [reply]
真的假的，笑死

**Comment by jiushi555** [comment]
我在想，在文革，你是不是就是那种举报亲妈的人

**Comment by zhaiyuxin103** [comment]
这就是你的信仰吗？不能有一丝一毫反对声音的出现？

**Comment by sengil** [comment]
你个大聪明，你是有多自卑张口闭口都是境外势力，矛盾还需要制造？

**Comment by SpockT** [comment]
脑残一个

**Comment by davidliu1645** [comment]
你他妈的是傻逼是吧

**Comment by yzyaz** [comment]
你能区分什么是国家, 什么是党派吗

**Comment by blacklion-spec** [comment]
我怀疑你是反串黑，😄

**Comment by toxin08x** [comment]
楼主和这个墙外的服务器应该互相算境外势力吧… 如果按照楼主的逻辑，楼主是不是应该滚回墙内？

**Comment by MYWmiss** [comment]
讲道理，断供的哪些人才是反华分子吧。法治国家，违法犯罪的事，保持零容忍。

**Comment by Joseph-Wheat-Carryier** [comment]
房子卖好好，公务员吃饱饱。
烂尾楼？关我屁事啊？

**Comment by SpockT** [comment]
本来就是政府导致的，为什么不能骂？

**Comment by fnnnnng** [comment]
大家快来看皇帝没急，这个阉人、太监急了

**Comment by lance111l** [comment]
支持  特朗普的账号都能封  这些一样能封
